run.sh

```diff
@@ -18,6 +18,6 @@ mkdir isp
 # Copy required files to isp directory
 cp ./prerequisites/.openapi-generator-ignore ./isp/.openapi-generator-ignore
 cp ./prerequisites/convenience._go ./isp/convenience.go
-
 docker build -t generate-sdk . --no-cache --build-arg OPENAPI_SPEC="${OPENAPI_SPEC}"
 docker run --rm -it -v ${SCRIPT_DIR}/isp:/go-sdk/isp generate-sdk
+cp ./prerequisites/client._go ./isp/client.go
```

Comment:
You can add that `client.go` should be ignored during re-generation in the [.openapi-generator-ignore](https://github.com/istreamlabs/go-sdk/blob/main/prerequisites/.openapi-generator-ignore). This would allow moving this line to above where the other `cp` commands to the `isp` directory are being made.